stallman/projeto-gnu.md

```diff
@@ -1,4 +1,60 @@
 [Voltar](intro.md)
 
 O projeto GNU
-====
\ No newline at end of file
+====
+
+<img alt="A mascote do GNU, criada por Aurelio A. Heckert" src="https://upload.wikimedia.org/wikipedia/en/thumb/2/22/Heckert_GNU_white.svg/1024px-Heckert_GNU_white.svg.png" width="150px" height="auto" />
+
+>*GNU's not Unix!*
+
+- ## O que é o projeto?
+
+O *projeto GNU* é um projeto colaborativo de Software Livre anunciado em 27 de Setembro de 1983 (Trabalhos nele apenas começaram em Janeiro de 1984).[^2] A iniciativa tem a intenção de **desenvolver e publicar** (colaborativamente) software para permitir que seus usuários tenham a liberdade de **utilizar, copiar, distribuir, estudar e modificá-lo**, com o objetivo final de dar **liberdade e controle aos usuários de computadores e seus dispositivos.** [^1]
+
+Dois anos depois que o sistema operacional foi anunciado, em Março de 1985, foi publicado o **Manifesto GNU**, pedindo apoio no desenvolvimento do OS GNU (Operational System GNU). [^3]
+
+Richard Stallman, acredita que, se um programa o agrada, ele deve compartilhá-lo com outras partes interessadas, e que se recusa a quebrar a solidariedade com outros usuários assim como fazem os vendedores de software. [^3]
+
+O projeto GNU é parte do *Movimento Software Livre*, uma campanha para a liberdade dos usuários de software. [^3]
+
+- ## Quais são seus objetivos?
+Richard Stallman criou o projeto GNU e junto do seu manifesto, o **Manifesto GNU**. Assim, podendo explicar suas razões e objetivos.
+No manifesto, Richard exprime a vontade de compartilhar os programas que gosta. Além disso, explica que abandonou seu trabalho no Laboratório de Inteligência Artificial do MIT, com objetivo de impedir que o MIT arrumasse alguma desculpa legal para impedí-lo de fornecer o GNU de forma livre. [^3]
```

Comment:
Eu acho que essa parte ficaria melhor logo no início do capítulo, pois fala, de início, que o Stallman foi o criador do projeto GNU.